reinforced_lib/agents/thompson_sampling.py

```diff
@@ -93,49 +123,50 @@ def update(state: AgentState, a: jnp.int32, r: jnp.int32, time: jnp.float32 = 0.
 
         Returns
         -------
-        out : AgentState
+        out : ThompsonSamplingState
             Updated agent state.
         """
 
         state = _decay_one(state, a, time)
-        state = AgentState(
+        state = ThompsonSamplingState(
             alpha=state.alpha.at[a].add(r),
             beta=state.beta.at[a].add(1 - r),
             last_decay=state.last_decay
         )
         return state
 
-    def sample(state: AgentState, key: jax.random.PRNGKey, time: jnp.float32 = 0.0) -> Tuple[jnp.float32, AgentState]:
+    def sample(state: ThompsonSamplingState, key: chex.PRNGKey,
+               time: chex.Scalar = 0.0) -> Tuple[ThompsonSamplingState, jnp.int32]:
         """
         Selects next action based on current agent state.
 
         Parameters
         ----------
-        state : AgentState
+        state : ThompsonSamplingState
             Current state of the agent.
-        key : jax.random.PRNGKey
+        key : chex.PRNGKey
             A PRNG key used as the random key.
         time : float
             Current time.
 
         Returns
         -------
-        out : Tuple[jnp.float32, AgentState]
+        out : Tuple[ThompsonSamplingState, int]
             Tuple containing selected action and updated agent state.
```

Comment:
Możesz zamienić na `"Tuple containing updated agent state and selected action"`, żeby kolejność opisu byłą zgodna z kolejnością w krotce